### PR TITLE
don't crash when scrollview's content is the same size

### PR DESCRIPTION
**Pull request by tshirtman**
also remove test that prevents scrolling when size_hint is not None,
that's not necessary.

There is still something that prevent scrolling when the size is exactly
the size of content, but that's rare enough if you don't set size_hint
to 1 in the direction you are interested in.

### CODE REVIEW

File: kivy/uix/scrollview.py [hunk 1]
```diff
@@ -286,7 +286,8 @@ def _set_do_scroll(self, value):
 
     :attr:`always_overscroll` is a
     :class:`~kivy.properties.BooleanProperty` and defaults to `True`.
-    versionadded:: 2.0.0
+
+    .. versionadded:: 2.0.0
```
Comment (matham) [inline kivy/uix/scrollview.py:290]: Don't you need a newline after this?

Comment (tshirtman) [reply kivy/uix/scrollview.py:290]: fixed
